packages/component/src/Attachment/TextContent.js

```diff
@@ -26,7 +33,13 @@ const TextContent = ({ contentType, text }) => {
   ) : (
     (text || '').split('\n').map((line, index) => (
       <React.Fragment key={index}>
-        <ScreenReaderText text={text} />
+        <ScreenReaderText
+          text={
+            remark()
```

Comment:
Is it possible to reuse the `remark` object from earlier? If not, we should probably put this in a variable instead of doing it inline. Also, we should probably memoize this.

Reply:
yes on the remark, I'll make a util. no on the memoize. xD